article/app/views/fragments/articleBody.scala.html

```diff
@@ -36,24 +36,47 @@
             }
 
             @if(amp) {
+
+                @fragments.mainMedia(article, amp)
+
                 @fragments.headTonal(article, model, isPaidContent, amp = amp)
 
-                @if(article.tags.isFeature && article.elements.hasShowcaseMainElement) {
-                    @fragments.mainMedia(article, amp)
-                }
             } else {
-                <div class="hide-on-mobile">
-                    @fragments.headTonal(article, model, isPaidContent, amp = amp)
+                @if(article.tags.isFeature && article.elements.hasShowcaseMainElement) {
 
-                    @if(article.tags.isFeature && article.elements.hasShowcaseMainElement) {
-                        @fragments.mainMedia(article, amp)
-                    }
-                </div>
-                <div class="mobile-only">
-                    @if(article.tags.isFeature && article.elements.hasShowcaseMainElement) {
-                        @fragments.mainMedia(article, amp)
-                    }
-                </div>
+                    <div class="content__header-container">
+                        @fragments.headTonal(article, model, isPaidContent)
+
+                        <div class="content__main-media-full-width-mobile">
+                            @fragments.mainMedia(article)
+                        </div>
+                    </div>
+
+                } else {
+                    <div class="content__header-container">
+                        @fragments.headTonal(article, model, isPaidContent)
+
+                        <div class="content__main-media-wrapper content__main-media-full-width-mobile">
+                            <div class="content__main-media">
+
+                                @fragments.mainMedia(article)
+                            </div>
+
+                            <div class="hide-on-mobile">
```

Comment:
TODO: I would like to find a way to only serve the content meta once (serving twice now), but I would rather have a think and do that in a second PR. I will ask @zeftilldeath about it when we are ready to tweak the desktop article design (if we do)